dev-python/pyxml/pyxml-0.8.4.recipe

```diff
@@ -1,20 +1,100 @@
-DESCRIPTION="Python XML package"
+SUMMARY="Python XML package"
+DESCRIPTION="Python XML comes with a validating XML parser, an implementation of the SAX \
+and DOM programming interfaces, an interface to the Expat parser (and the Expat parser \
+itself), and a C helper module that can speed up xmllib.py by a factor of 5."
 HOMEPAGE="http://pyxml.sourceforge.net/topics/index.html"
-SOURCE_URI="http://sourceforge.net/projects/pyxml/files/latest/download?source=files"
-CHECKSUM_MD5="1f7655050cebbb664db976405fdba209"
+COPYRIGHT="2000 Fourthought Inc
+	1991 - 1995, Stichting Mathematisch Centrum Amsterdam
+	Greg Stein
```

Comment:
```suggestion
```

Being in public domain means that the author has waived their copyright, so we shouldn't specify them here.